recipes/miktex-portable/post-link.bat

```diff
@@ -0,0 +1,6 @@
+rem latex packages which are needed by nbconvert and other pdf producers
+%PREFIX%\Library\miktex-portable\miktex\bin\mpm --update-db && if errorlevel 1 exit 1
+for %%x in (adjustbox booktabs collectbox fancyvrb ifoddpage mptopdf ucs url) do (
+	%PREFIX%\Library\miktex-portable\miktex\bin\mpm --install %%x  && if errorlevel 1 exit 1
+)
```

Comment:
Note that this might break the package if the latex package repository decides to drop/rename one of these packages. It seems unlikely but it happened to us more than once in the course of a single year.